changelogs/unreleased/4317-syntax-error-when-checking-if-dict-value-is-defined.yml

```diff
@@ -0,0 +1,6 @@
+description: Fix syntax error when calling "is defined" on dictionary lookup
+change-type: patch
+destination-branches: [master]
```

Comment:
Should it go into iso5 as well ? Or is it not wise to squeeze this in before the release ?

Reply:
It should definitely go into iso5. If we would want to exclude it from the release we'd just have to hold off on merging it for now. But I think it's safe to include it. @wouterdb do you agree?

Reply:
i do